domain/src/main/kotlin/no/nav/su/se/bakover/domain/Sak.kt

```diff
@@ -198,7 +199,9 @@ data class Sak(
                     .reduser()
             }.reduser()
     }
-    fun hentÅpneKlager(): List<Klage> = klager.filter { it.erÅpen() }
+
+    /** Skal ikke kunne ha mer enn én åpen klage av gangen. */
+    fun kanOppretteKlage(): Boolean = klager.none { it.erÅpen() }
```

Comment:
Hvis man heller kaller metoden det den faktisk gjør (feks `åpenKlageEksisterer`) så kan kan fjerne slike kommentarer og lese betingelsen direkte i koden. Selvdokumenterende kode altså 🤠

Reply:
I mange tilfeller er jeg enig i slik navngivning, men akkurat her ønsket jeg å abstrahere bort den underliggende logikken. Men man kan jo spørre seg hvorfor ikke funksjonen heller er erstattet med en `opprettKlage(): Either<KunneIkkeOppretteKlage,OpprettetKlage>`. Siden saken bør stå ansvarlig for å bestemme om man kan gjøre slike operasjoner som `opprett klage`, `opprett behandling`, `opprett revurdering`, `stans og gjenoppta`, `Ny kontrollsamtale`. Da vil vi ha en enhetlig måte å gjøre disse operasjonene på.